pandas/tests/test_panel.py

```diff
@@ -152,11 +153,15 @@ def _check_stat_op(self, name, alternative, obj=None, has_skipna=True):
 
         if has_skipna:
 
-            def skipna_wrapper(x):
-                nona = remove_na_arraylike(x)
-                if len(nona) == 0:
-                    return np.nan
-                return alternative(nona)
+            if skipna_alternative:
```

Comment:
maye should move this skipna_wrapper to ``pandas/util/testing.py`` as its in 3 places now